src/main/java/hudson/remoting/Channel.java

```diff
@@ -832,6 +854,11 @@ public void setJarCache(@Nonnull JarCache jarCache) {
      */
     public <V,T extends Throwable>
     V call(Callable<V,T> callable) throws IOException, T, InterruptedException {
+        if (isClosingOrClosed()) {
+            // No reason to even try performing a user request
+            throw new IOException("Remote call on " + name + " failed. The channel is closing down or has closed down");
```

Comment:
Not exactly. `ChannelClosedException` would have been thrown if the channel is closed, but not "is closing". Probably there is no difference from user perspective though